lib/PersistHelper.ts

```diff
@@ -554,6 +561,26 @@ export const retrieveUserAccessTokenAsync = async (
     return data.accessToken;
 };
 
+export const retrieveAllUsersAccessTokenAsync = async (
+    read: IRead,
+): Promise<Array<string> | null> => {
+    const associations: Array<RocketChatAssociationRecord> = [
+        new RocketChatAssociationRecord(
+            RocketChatAssociationModel.MISC,
+            MiscKeys.UserRegistration
+        ),
+    ];
+
+    const persistenceRead: IPersistenceRead = read.getPersistenceReader();
+    const results = await persistenceRead.readByAssociations(associations) as Array<UserRegistrationModel>;
+
+    if (results === undefined || results === null || results.length == 0) {
```

Comment:
```suggestion
    if (results == null || results.length == 0) {
```

undefined == null is true